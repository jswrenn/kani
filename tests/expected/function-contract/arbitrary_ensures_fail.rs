// Copyright Kani Contributors
// SPDX-License-Identifier: Apache-2.0 OR MIT

#[kani::ensures(result == x)]
fn max(x: u32, y: u32) -> u32 {
    if x > y { x } else { y }
}

#[kani::proof]
fn main() {
    max(kani::any(), kani::any());
}
