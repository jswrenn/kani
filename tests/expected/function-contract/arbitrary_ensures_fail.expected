arbitrary_ensures_fail::max.missing_definition \
	 - Status: FAILURE \
	 - Description: "Function `arbitrary_ensures_fail::max` with missing definition is unreachable" \
